lib/rsolr/client.rb

```diff
@@ -63,6 +65,11 @@ def paginate page, per_page, path, opts = nil
     execute build_paginated_request(page, per_page, path, opts)
   end
   
+  # Sends a query to /select or the custom configured select handler.
+  def select opts={}
```

Comment:
I don't see the need for this. Why not just call `get '/select'` or whatever handler you want to use here. the `/select` handler is not something you have to use, so RSolr has no need to know about this.